### DISCUSSION TITLE
RemoteApp sometimes doesn't show on the client

### DISCUSSION THREAD

**Discussion by LeighWardle** [kimmknight/remoteapptool#117] [General]
Hi Everyone,

We have used RemoteApp tool for many years.
We have just moved our main app (a MS Access database) from an AWS EC2 instance to Azure.

Sometimes the RemoteApp doesn't show on the client (our suspicion is that it is open on the server).
This never happened on the old server.

Does anyone have a clue as to what may be happening?

Regards,
Leigh

**Comment by LeighWardle** [comment]
Hi Everyone,

We have given up using RemoteApp, as we cannot figure out why it is so unreliable for us.

Regards,
Leigh

**Comment by MrBrianGale** [comment]
That is not a lot of time for support on a FREE tool. 3 days and you drop the tool?
Now, that being said, you provided very little for us to go on. No logs, no description of your environment, no description on tools being used... nothing for us to go on.

Now, with the above being said, the remoteapptool is mostly a configuration tool for using the RDP protocol for remoteapps. This tool does things in a VERY similar manner to how Windows does it on Windows Server.

Now, the only reasons I am aware of for an app not "starting" when run as a remoteapp are:
1- not enough server CALs so users are being disconnected when starting (common when using a desktop OS)
2- app crashes and doesn't have proper error handling for a remoteapp session
3- server system configured incorrectly

My suspicion is that it is case number 2 or 3 that is your problem. Are you 100% sure you set everything up correctly on the server? And have you reviewed the logs (windows and application, RemoteAppTool is not the culprit here, I am 100% confident on that).

IF you are 100% confident everything is set up correctly on your end, we will need a LOT more to go on. For example, what was the old OS and new OS (version and edition)? What are the client systems using for RDP? What entries are in the logs at time of failure on the server machine and client machine?
